store/src/app/products/components/cart/cart.component.html

```diff
@@ -0,0 +1,29 @@
+<ng-container *ngIf="products.length >= 1">
+  <div class="container">
+    <h1 class="title">Cart</h1>
+    <p>Total: {{totalPrice | hryvniaPipe}}</p>
+    <div class="products-cards">
+      <app-product-card [quantityButtons]="true" *ngFor="let item of products" [product]="item"
```

Comment:
Better to format parameters like this, to increase readability

![image](https://user-images.githubusercontent.com/43652702/209469244-567afcb4-4bfc-4309-abda-2e126fe0e882.png)

Also mention, that structural directives like `*ngFor`, `*ngIf` should be first.

Reply:
Done